src/lib/4337.js

```diff
@@ -0,0 +1,249 @@
+import { ethers } from "ethers";
+import tokenEntryPointContractAbi from "smartcontracts/build/contracts/tokenEntryPoint/TokenEntryPoint.abi";
+import accountFactoryContractAbi from "smartcontracts/build/contracts/accfactory/AccountFactory.abi";
+import accountContractAbi from "smartcontracts/build/contracts/account/Account.abi";
+import tokenContractAbi from "smartcontracts/build/contracts/erc20/ERC20.abi";
```

Comment:
it's weird to have this `.abi` extension. It's a json file. Would love to move this to `.abi.json` at some point.